src/contexts/RDKitContext.tsx

```diff
@@ -24,7 +28,7 @@ export const RDKitProvider: React.FC<PropsWithChildren> = ({ children }) => {
     return () => {
       isProviderMounted = false;
     };
-  }, []);
+  }, [RDKit]);
```

Comment:
we can avoid having RDKit in the dependency (as we have a setRDKit in the useEffect) by doing  
```ts
    if (!rdkitInstance && globalThis.initRDKitModule) {
   ```
and maybe we can rename `rdkitInstance` to `initialRdkitInstance` to avoid confusion.
@atfera  what do you think?

Reply:
Done